app/models/data-operation.ts

```diff
@@ -231,6 +233,15 @@ export default class DataOperation {
     this.decrypter = decrypter;
   }
 
+  /**
+   * Returns true if the operation is requesting subsetting
+   *
+   * @returns true if the operation requests subsetting
+   */
+  get shouldSubset(): boolean {
```

Comment:
This seems too generic to me. I think we need different subset parameters - variableSubset, temporalSubset, spatialSubset, etc.